src/git.ts

```diff
@@ -46,53 +46,83 @@ const processKillTriggerWords = [
  * @param input string input from stdio
  */
 export const testForKill = (input: string) => {
-  for (const triggerWord in processKillTriggerWords) {
-    if (input.includes(processKillTriggerWords[triggerWord])) {
-      return true;
+  for (const wordIndex in processKillTriggerWords) {
+    if (input.includes(processKillTriggerWords[wordIndex])) {
+      return processKillTriggerWords[wordIndex];
```

Comment:
This may be better as [for...of](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of)

Reply:
Good shout. Updated!